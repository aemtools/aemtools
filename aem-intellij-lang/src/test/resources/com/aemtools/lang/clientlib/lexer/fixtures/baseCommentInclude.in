#base=dir
# this is comment
file.js