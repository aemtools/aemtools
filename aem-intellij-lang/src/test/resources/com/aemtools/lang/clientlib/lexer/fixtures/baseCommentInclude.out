[Cd] #base= ('#base=')
[Cd] WORD ('dir')
WHITE_SPACE ('\n')
[Cd] COMMENT_TOKEN ('# this is comment')
WHITE_SPACE ('\n')
[Cd] WORD ('file.js')
